src/pygambit/behav.pxi

```diff
@@ -376,6 +376,24 @@ class MixedBehaviorProfile:
             raise ValueError("action_value() is not defined for the chance player")
         return self._action_value(resolved_action)
 
+    def realiz_prob(self, node: typing.Union[Node, str]):
+        """Returns the probability with which an node is reached.
+
+        Parameters
+        ----------
+        node : Node or str
+            The node to get the payoff for.  If a string is passed, the
+            node is determined by finding the node with that label, if any.
+
+        Raises
+        ------
+        MismatchError
+            If `node` is an `Node` from a different game.
+        KeyError
+            If `node` is a string and no node in the game has that label.
+        """
+        return self._realiz_prob(self.game._resolve_node(node, 'node_prob'))
```

Comment:
The name of the function is `realiz_prob` so we should pass `'realiz_prob'` to `._resolve_node`, instead of `'node_prob`'

Reply:
Done